.tekton/release-service-catalog-pull-request.yaml

```diff
@@ -18,7 +18,8 @@ metadata:
       || "schema/dataKeys.json".pathChanged()
       || "stepactions/*/*.yaml".pathChanged()
       || "pipelines/internal/*/*.yaml".pathChanged()
-      || "integration-tests/*".pathChanged())
+      || "integration-tests/*".pathChanged()
```

Comment:
Maybe add the paths of collectors' task and pipeline? If any change about collectors' task or pipeline, it need to trigger the integration test. Right? Or use a seperate PR to add these paths?